orbit/cmd/desktop/desktop.go

```diff
@@ -466,10 +453,40 @@ func setupLogs() {
 		MaxAge:     28, // days
 	}
 
-	log.Logger = log.Output(zerolog.MultiLevelWriter(
-		zerolog.ConsoleWriter{Out: logFile, TimeFormat: time.RFC3339Nano, NoColor: true},
-		stderrOut,
-	))
+	consoleWriter := zerolog.ConsoleWriter{Out: logFile, TimeFormat: time.RFC3339Nano, NoColor: true}
+	log.Logger = log.Output(consoleWriter)
+}
+
+// setupStderr redirects stderr output to a file.
+func setupStderr() {
+	dir, err := logDir()
+	if err != nil {
+		log.Error().Err(err).Msg("find directory for stderr")
+		return
+	}
+
+	if err := os.MkdirAll(dir, 0o755); err != nil {
+		log.Error().Err(err).Msg("make directories for stderr")
+		return
+	}
+
+	stderrFile, err := os.OpenFile(filepath.Join(dir, "Fleet", "fleet-desktop.err"), os.O_WRONLY|os.O_CREATE|os.O_TRUNC, 0o666)
```

Comment:
Shouldn't we use `lumberjack.Logger` here to get log rotation, etc.
And why not direct stderr to `fleet-desktop.log` instead of making a separate file?

Reply:
I didn't want two packages to write to the same file. (Maybe both are making assumptions that they are the only entity writing to such file.)